crates/bevy_scene/src/scene.rs

```diff
@@ -1,20 +1,21 @@
 use bevy_app::AppTypeRegistry;
+use bevy_derive::{Deref, DerefMut};
 use bevy_ecs::{
     entity::EntityMap,
     reflect::{ReflectComponent, ReflectMapEntities},
     world::World,
 };
 use bevy_reflect::TypeUuid;
 
-use crate::{InstanceInfo, SceneSpawnError};
+use crate::{DynamicScene, InstanceInfo, SceneSpawnError};
 
 /// To spawn a scene, you can use either:
 /// * [`SceneSpawner::spawn`](crate::SceneSpawner::spawn)
 /// * adding the [`SceneBundle`](crate::SceneBundle) to an entity
 /// * adding the [`Handle<Scene>`](bevy_asset::Handle) to an entity (the scene will only be
 /// visible if the entity already has [`Transform`](bevy_transform::components::Transform) and
 /// [`GlobalTransform`](bevy_transform::components::GlobalTransform) components)
-#[derive(Debug, TypeUuid)]
+#[derive(Debug, TypeUuid, Deref, DerefMut)]
```

Comment:
I'm reluctant to add these derives; they make things more implicit and are likely to break once we need to add more fields.

Reply:
IMO the Deref / DerefMut impls on Scene should be their own PR so they get appropriate review attention.

Reply:
> 'm reluctant to add these derives; they make things more implicit and are likely to break once we need to add more fields.

Then let's remove them later *if* we need to add more fields? For now they make working with scenes a lot nicer.

Reply:
Hmm, I'm fine to leave it. We should write a manual Deref / DerefMut impl if and when we add more fields then, and call this out in the docs though.

Reply:
Personally, I don't think the `Deref`/`DerefMut` derives are needed. It's probably better for users to explicitly be aware that what they're working with is basically just a plain ol' `World`. And yeah, it makes future additions to `Scenes` less problematic (e.g. versioning, etc.).